crates/re_types/definitions/rerun/archetypes/boxes3d.fbs

```diff
@@ -11,42 +11,8 @@ namespace rerun.archetypes;
 
 /// A batch of 3d boxes with half-extents and optional center, rotations, rotations, colors etc.
 ///
-/// \py Example
-/// \py -------
-/// \py
-/// \py Simple 3D boxes:
-/// \py ```python
-/// \py \include:../../../../../docs/code-examples/box3d_simple.py
-/// \py ```
-/// \py
-/// \py Batch of 3D boxes:
-/// \py ```python
-/// \py \include:../../../../../docs/code-examples/box3d_batch.py
-/// \py ```
-///
-/// \rs ## Example
-/// \rs
-/// \rs Simple 3D boxes:
-/// \rs ```ignore
-/// \rs \include:../../../../../docs/code-examples/box3d_simple.rs
-/// \rs ```
-/// \rs
-/// \rs Batch of 3D boxes:
-/// \rs ```ignore
-/// \rs \include:../../../../../docs/code-examples/box3d_batch.rs
-/// \rs ```
-///
-/// \cpp ## Example
-/// \cpp
-/// \cpp Simple 3D boxes:
-/// \cpp ```
-/// \cpp \include:../../../../../docs/code-examples/box3d_simple.cpp
-/// \cpp ```
-/// \cpp
-/// \cpp Batch of 3D boxes:
-/// \cpp ```
-/// \cpp \include:../../../../../docs/code-examples/box3d_batch.cpp
-/// \cpp ```
+/// \example box3d_simple
+/// \example box3d_batch
```

Comment:
This misses out on the titles (`Simple 3D boxes` and `Batch of 3D boxes`). I suggest:

```suggestion
/// \example box3d_simple "Simple 3D boxes"
/// \example box3d_batch "Batch of 3D boxes"
```